### PR TITLE
Added examples for map and slice

### PR DESCRIPTION
**Pull request by MichaelS11**

### CODE REVIEW

File: vm/example_containers_test.go [hunk 1]
```diff
@@ -69,6 +77,43 @@ println(c)
...
+	if err != nil {
+		log.Fatalf("Define error: %v\n", err)
+	}
+
+	script := `
+a = {}
+println(a)
+
+a.b = 1
+println(a)
```
Comment (Chyroc) [inline vm/example_containers_test.go]: add `println(a['b'])`?

File: vm/example_containers_test.go [hunk 2]
```diff
@@ -69,6 +77,43 @@ println(c)
...
+
+a = {}
+a["b"] = 1
+println(a)
```
Comment (Chyroc) [inline vm/example_containers_test.go]: add `println(a.b)`?

File: vm/example_containers_test.go [hunk 3]
```diff
@@ -69,6 +77,43 @@ println(c)
...
+
+println(len(a))
```
Comment (Chyroc) [inline vm/example_containers_test.go]: add 
```
a = {}
println(a.c)
```
?

File: vm/example_containers_test.go [hunk 4]
```diff
@@ -69,6 +77,43 @@ println(c)
...
+`
```
Comment (Chyroc) [inline vm/example_containers_test.go]: add 
```
a = {'a': 1}
delete(a, 'a')
```
?

**Comment by MichaelS11** [comment]
Updated with more examples.

Not expecting to cover all the examples in each PR. Just adding handful or two at a time.

File: vm/example_operators_test.go [hunk 5]
```diff
@@ -176,3 +176,41 @@ println(a)
...
+	// output:
+	// bc
+	// ab
+	// b
+	//
+	// [2 3]
+	// [1 2]
+	// [2]
+
+}
```
Comment (mattn) [inline vm/example_operators_test.go]: probably this file is not gofmt-ed?

Comment (MichaelS11) [reply vm/example_operators_test.go]: Fixed

**Comment by mattn** [comment]
Thank you

**Pull request merged**